tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -366,9 +370,9 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 				output.append('''// TODO interaction type '«interaction.name»' does not have a proper method reference''')
 			}
 		} else if (step.componentContext != null) {
-			output.append('''// TODO could not resolve '«step.componentContext.component.name»' - «stepLog»''')
+			output.append('''fail("Template '«stepLog»' cannot be resolved with any known macro/fixture. Please check your file '«testCase.name»' at line «NodeModelUtils.findActualNodeFor(step).startLine»");''')
```

Comment:
since '<<testcase.name>>' is no file, the error message should be something like 'Please check your test case ='...' / macro = '...' / configuration = '...' using the full qualified name'. Producing this message could be extracted into a separate method.